test/base_delayed_paperclip_test.rb

```diff
@@ -1,254 +0,0 @@
-###
-#  DEPRECATED, Don't Use
-###
-
-module BaseDelayedPaperclipTest
```

Comment:
Necessary to delete all of these?  Worried we haven't covered all of them in the rspec tests